### PR TITLE
did some code clean_up in some component

### PR DESCRIPTION
**Pull request by Tijani-zainab**
did some code cleanup in some component, getting rid of some annoying vscode errors

…still working on moving the rest of the stylesheet to styled component, pr in batch for easier review!

### CODE REVIEW

File: client/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -15,4 +15,6 @@ const Footer = () => (
   </div>
 );
 
+// export default Footer;
+
```
Comment (jazhen) [inline client/src/components/Footer/Footer.js:19]: Can you please remove this commented out line?

**Review by jazhen — CHANGES_REQUESTED** [review]
Great work! Can you just fix one minor issue?

File: client/src/components/Footer/Footer.js (hunk 1 above)
Comment (Tijani-zainab) [reply client/src/components/Footer/Footer.js:19]: Thank you!
 @jazhen  I actually wrote that piece of code trying to get rid of the `index.js` file and export default directly from the `footer.js` file and fix all imports where necessary. for some reason that broke the app even after fixing imports in other places.
water

I commented it out to get back to it later.
find out what the error is, see if I can fix then get rid of the `index.js` file

Comment (Tijani-zainab) [reply client/src/components/Footer/Footer.js:19]: but wait, when you say remove you meant get rid of it ? 

looks like I might be unclear what you meant, if it's to get rid of it. will do that
but if it's to uncomment it, that might cause some issues.

Comment (jazhen) [reply client/src/components/Footer/Footer.js]: Yup, just meant to get rid of it. Just wanted to make sure we don't have unused or in progress code committed if possible. Thank you!

**Review by jazhen — APPROVED** [review]
Good job on your first commit @Tijani-zainab . Thanks!

**Comment by Tijani-zainab** [comment]
Thank you! @jazhen

**Pull request merged**